crates/bevy-inspector-egui/src/quick.rs

```diff
@@ -338,6 +339,7 @@ impl<A: Asset + Reflect> Plugin for AssetInspectorPlugin<A> {
         check_default_plugins(app, "AssetInspectorPlugin");
 
         if !app.is_plugin_added::<DefaultInspectorConfigPlugin>() {
+            println!("LMAO");
```

Comment:
LMAO

Reply:
Yeah it needs a pass more before getting merged xD